XCUITests/FirstRunTourTests.swift

```diff
@@ -42,30 +42,6 @@ class FirstRunTourTests: BaseTestCase {
         waitForExistence(topSites)
     }
 
-    func testStartBrowsingFromSecondScreen() {
```

Comment:
I think we should leave this test to be sure we can start browsing from the sync screen...what do you think @SimonBasca? Thanks!